apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.html.ts

```diff
@@ -0,0 +1,674 @@
+import { html, escapeHtml } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+import { nodeModulesAssetPath } from '../../lib/assets';
+import { GroupConfigSchema, IdSchema, UserSchema } from '../../lib/db-types';
+import { z } from 'zod';
+
+export const GroupConfigInfoSchema = GroupConfigSchema.extend({
+  defaultMin: z.number().optional(),
+  defaultMax: z.number().optional(),
+});
+type GroupConfigInfo = z.infer<typeof GroupConfigInfoSchema>;
```

Comment:
I know you're copying the original behaviour, but I think extending the schema only to set these values does not actually make sense here. I think it makes more sense to move the `defaultMin`/`defaultMax` calculation to an expression in line 212, or a separate variable.

Reply:
That makes sense. Also, this is being used in one of the modals which we have now moved to a separate function. I have now moved the calculation of min/max group members to the call of the `autoAssessmentGroupsModal` and removed the additional schema.